src/Utils/useVoiceRecorder.ts

```
@@ -0,0 +1,123 @@
+import { useEffect, useState } from "react";
+import { Error } from "./Notifications";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)